### ISSUE TITLE
Error in getAttestationAttesterCount() attestation.aggregationBits = null

### ISSUE THREAD

**Issue by dapplion** [ChainSafe/lodestar#2283]
**Describe the bug**

In some sim test multi-node multi-thread run the node error repeatedly with 

```
2021-03-28 11:28:38 [NODE 2 NETWORK]  [31merror[39m: Gossip attestation validation threw a non-AttestationError message=Cannot convert undefined or null to object
TypeError: Cannot convert undefined or null to object
    at Function.from (<anonymous>)
    at getAttestationAttesterCount (/home/runner/work/lodestar/lodestar/packages/lodestar/src/chain/validation/attestation.ts:163:16)
    at Object.validateGossipAttestation (/home/runner/work/lodestar/lodestar/packages/lodestar/src/chain/validation/attestation.ts:17:19)
    at validateCommitteeAttestation (/home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/validatorFns/attestation.ts:23:11)
    at /home/runner/work/lodestar/lodestar/packages/lodestar/src/network/gossip/validator.ts:44:11
    at Eth2Gossipsub.validate (/home/runner/work/lodestar/lodestar/node_modules/libp2p-interfaces/src/pubsub/index.js:594:13)
    at Eth2Gossipsub.<anonymous> (/home/runner/work/lodestar/lodestar/node_modules/libp2p-gossipsub/src/index.js:328:39)
    at Generator.next (<anonymous>)
    at /home/runner/work/lodestar/lodestar/node_modules/libp2p-gossipsub/src/index.js:27:71
    at new Promise (<anonymous>)
```

The error is thrown in this function

https://github.com/ChainSafe/lodestar/blob/a20c540d0f585107a5952cdc4cc0a45810c922f7/packages/lodestar/src/chain/validation/attestation.ts#L162-L164

**Expected behavior**

Should never throw this

**Steps to Reproduce**

Run the sim tests, here are full debug logs for that run

[multinode-multithread-test-logs.txt](https://github.com/ChainSafe/lodestar/files/6217260/multinode-multithread-test-logs.txt)

**Comment by twoeths** [comment]
it's good that we can reproduce in e2e bc I saw the same issue when I sync Prater

```
2021-03-28 01:56:46 [NETWORK]         error: Gossip attestation validation threw a non-AttestationError message=Cannot convert undefined or null to object
TypeError: Cannot convert undefined or null to object
    at Function.from (<anonymous>)
    at getAttestationAttesterCount (/root/lodestar/packages/lodestar/src/chain/validation/attestation.ts:163:16)
```

**Comment by wemeetagain** [comment]
Possibly resolved in #2289 ?

**Comment by twoeths** [comment]
it's still an issue with ssz 0.8.0
